src/frontend/komponenter/Fagsak/Vilkårsvurdering/Registeropplysninger.tsx

```diff
@@ -26,20 +26,12 @@ interface IRegisteropplysningerProps {
 }
 
 const Registeropplysninger: React.FC<IRegisteropplysningerProps> = ({ opplysninger }) => {
-    const finnesTomPeriode = (opplysninger: IRestRegisteropplysning[]): boolean =>
-        !!opplysninger.find(opplysning => !opplysning.fom && !opplysning.tom);
-    const finnesTomPeriodePåPerson =
-        finnesTomPeriode(opplysninger.oppholdstillatelse) ||
-        opplysninger.oppholdstillatelse.length === 0 ||
-        finnesTomPeriode(opplysninger.statsborgerskap) ||
-        opplysninger.statsborgerskap.length === 0 ||
-        finnesTomPeriode(opplysninger.bostedsadresse) ||
-        opplysninger.bostedsadresse.length === 0;
+    const bleOpprettetUtenRegisteropplysninger = opplysninger.statsborgerskap.length === 0;
```

Comment:
Bare for å være sikker, blir det en tom liste eller kan det også bli null om statsborgerskap ikke har blitt satt?